templates/cloudflare-workers/package.json

```diff
@@ -3,31 +3,30 @@
   "sideEffects": false,
   "scripts": {
     "build": "remix build",
-    "deploy": "wrangler publish",
-    "dev:remix": "remix watch",
-    "dev:miniflare": "cross-env NODE_ENV=development miniflare ./build/index.js --watch",
-    "dev": "remix build && run-p \"dev:*\"",
-    "start": "cross-env NODE_ENV=production miniflare ./build/index.js"
+    "dev": "npm run build && concurrently \"npm:dev:remix\" \"npm:dev:cloudflare\"",
+    "dev:remix": "cross-env NODE_ENV=development remix watch",
+    "dev:cloudflare": "cross-env NODE_ENV=development wrangler dev --local --config wrangler.dev.toml",
+    "start": "npm run dev:cloudflare"
   },
   "dependencies": {
+    "@cloudflare/kv-asset-handler": "^0.2.0",
     "@remix-run/cloudflare": "*",
-    "@remix-run/cloudflare-workers": "*",
     "@remix-run/react": "*",
-    "cross-env": "^7.0.3",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
```

Comment:
Does this mean, we can update the `cloudflare-pages` template to React 18 as well?